### PR TITLE
[Calamari]Update deps job for docker image build, readme, pr template

### PR DESCRIPTION
**Pull request by Dengjianping**
1. Update deps job for docker image build.
2. update readme.
3. add pr template.

### CODE REVIEW

**Review by stechu — APPROVED** [review]

**Review by grenade — APPROVED** [review]
lgtm

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
+   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
+- [ ] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
```
Comment (ghzlatarev) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: Not sure if this template will do anything.
It needs to be in the default branch,

Comment (ghzlatarev) [reply .github/PULL_REQUEST_TEMPLATE.md:31]: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#:~:text=You%20must%20create%20templates%20on%20the%20repository%27s%20default%20branch

Comment (stechu) [reply .github/PULL_REQUEST_TEMPLATE.md:31]: @ghzlatarev now the manta-pc is the default branch

**Comment by ghzlatarev** [comment]
@Dengjianping can you please add `Verify benchmarks & weights have been updated for any modified runtime logics.` as a bullet in the PR template ?

**Pull request closed without merging**